### PR TITLE
fixing the description of geography_areas and top_level_grographies

### PR DESCRIPTION
**Pull request by davidplumb**

### CODE REVIEW

**Comment by davidplumb** [comment]
the sql on this is wrong because the other PR addresses it, reminder here to make sure one is merged into the other before the second merge or it will cause potential overwriting of the formatting

File: tables/geography_areas.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+|2006|LA|Local Authorities|
+|2007|WED|Wards and Electoral Divisions|
+|2008|MSOAIZ|Middle Super Output Areas and Intermediate Zones|
+|2009|LSOADZ|Lower Super Output Areas and Data Zones|
+|2010|OASA|Output Areas and Small Areas|
+|2011|MLA|Merging Local Authorities|
+|2012|MWED|Merging Wards and Electoral Divisions|
+|2013|WZLYR|Workplace Zone Layer|
+
+If you combine the two values then you get the resulting `geography_area` which is referenced in the tables [topic_combinations](topic_combinations.md) as `geography_combinations` and [variable_combinations](variable_combinations.md) as `geography_combination`.The data is stored with the format of: ${geography_grouping_id}:${top_level_geography_id} e.g. 2006:4 (which in this case represents the isle of Wight local authority).
```
Comment (tomgiddingsjisc) [inline tables/geography_areas.md]: isle > Isle

File: tables/geography_areas.md [hunk 2]
```diff
@@ -31,17 +63,15 @@ SELECT geography_areas.id,
...
 
 |column|type|use|
 |-|-|-|
 |id|int4|Primary key.|
-|top_level_geography_id|int4|Foreign key for `top_level_geographies`.|
+|top_level_geography_id|int4|Foreign key for [top_level_geographies](top_level_geographies.md) (see the `What are geography areas` section for how this relatest to external tables).|
 |geography_grouping_id|int4|Relates to a geographical group in `geography_groupings`, eg. `Gwynedd` has a value of `2,006` which relates to the grouping of `Local Authorities`.|
 |description|varchar(255)|An end-user friendly description for the geography.|
-|geography_code|varchar(50)|A unique identifier for a geography.|
+|geography_code|varchar(50)|foregin key for [geography_groupings](geography_groupings.md) (see the `What are geography areas` section for how this relatest to external tables)|
```
Comment (tomgiddingsjisc) [inline tables/geography_areas.md]: foregin > foreign
relatest > relates

File: tables/top_level_geographies.md [hunk 3]
```diff
@@ -21,7 +21,26 @@ A `top-level geography` refers to the first (highest) geography level selectable
...
+|2006|LA|Local Authorities|
+|2007|WED|Wards and Electoral Divisions|
+|2008|MSOAIZ|Middle Super Output Areas and Intermediate Zones|
+|2009|LSOADZ|Lower Super Output Areas and Data Zones|
+|2010|OASA|Output Areas and Small Areas|
+|2011|MLA|Merging Local Authorities|
+|2012|MWED|Merging Wards and Electoral Divisions|
+|2013|WZLYR|Workplace Zone Layer|
+
+If you combine the two values then you get the resulting `geography_area` which is referenced in the tables [geography_areas](geography_areas.md)[topic_combinations](topic_combinations.md) as `geography_combinations` and [variable_combinations](variable_combinations.md) as `geography_combination`.The data in [topic_combinations](topic_combinations.md) and [variable_combinations](variable_combinations.md) is stored with the format of: ${geography_grouping_id}:${top_level_geography_id} e.g. 2006:4 (which in this case represents the isle of Wight local authority). While [geography_areas](geography_areas.md) gives a description on what the areas represent.
```
Comment (tomgiddingsjisc) [inline tables/top_level_geographies.md]: isle > Isle

**Review by tomgiddingsjisc — APPROVED** [review]

**Pull request merged**